src/components/MoviesList/MoviesList.tsx

```diff
@@ -4,12 +4,22 @@ import { MovieCard } from '../MovieCard';
 
 interface Props {
   movies: Movie[];
+  query: string;
 }
```

Comment:
`MovieCard` should know nothing about query, it's another abstraction level
let's keep matching query flow one level above